spec/lib/tasks/evm_application_spec.rb

```diff
@@ -94,4 +94,27 @@
       end
     end
   end
+
+  describe ".deployment_status" do
+    let!(:server) { EvmSpecHelper.local_miq_server }
+
+    it "returns new_deployment if the database is not migrated" do
+      expect(ActiveRecord::Migrator).to receive(:current_version).and_return(0)
+      expect(described_class.deployment_status).to eq("new_deployment")
+    end
+
+    it "returns new_replica if the current server is not seeded" do
+      server.delete
```

Comment:
It feels awkward to create the server then delete it.  I'd rather see the server created only in the test cases where it is needed.